### PR TITLE
Lint remaining files

### PR DESCRIPTION
**Pull request by ahuber21**
* Lint remaining files
* Limit string length in copyright disclaimer to 80 cols

### CODE REVIEW

File: gen.py [hunk 1]
```diff
@@ -32,13 +33,19 @@
...
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    argParser.add_argument('--daalroot', required=True,
-                           help="DAAL root directory (reads include dir in there)")
-    argParser.add_argument('--outdir', default='build',
-                           help="Output directory to store wrapper files to")
-    argParser.add_argument('--wall', default=False,
-                           action='store_true', help="Emit all warnings")
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    argParser.add_argument(
```
Comment (Alexsandruss) [inline gen.py:38]: @ahuber21, doesn't linter change camel to snake case for variables? I thought it's basic requirement for python linter

Comment (ahuber21) [reply gen.py:38]: That's new to me. Which linter is able to do that? Maybe we could add it to the pipeline

**Comment by napetrov** [comment]
@Alexsandruss  @KulikovNikita  @samir-nasibli  - are we going to merge this?

**Comment by Alexsandruss** [comment]
@mergifyio rebase

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**